### ISSUE TITLE
The mistake: wrong name for the train set and dev set for UP_Portuguese-Bosque

### ISSUE THREAD

**Issue by scofield7419** [System-T/UniversalPropositions#4]
You may  wrongly named the train set file and dev set file for _UP_Portuguese-Bosque_ by mistake.
The two file names need to be exchanged.

**Comment by arademaker** [comment]
This issue is solved but many problems were found during the merge of the current UD data from the master branch of http://github.com/universaldependencies/UD_Portuguese-Bosque and the SRL annotations in the https://github.com/System-T/UniversalPropositions/tree/master/UP_Portuguese-Bosque. 

See https://github.com/System-T/UniversalPropositions/blob/issue-4/UP_Portuguese-Bosque/README.md#merging-up-and-ud-datasets

We will need to manually inspect some sentences and decide about the final format of the files in this repository. To make Portuguese data similar to the remain files, I will temporarily convert the new CoNLL-U valid files to CoNLL files (one extra column for each predicate).

**Comment by arademaker** [comment]
@huaiyu-zhu comments are welcome.

**Comment by arademaker** [comment]
I found at least one sentence in the current set of PT sentences in the MASTER branch with a wrong `sent_id`. Fixed in https://github.com/System-T/UniversalPropositions/commit/109816fe866fe235f332338b0d5953ccc082cc27

**Comment by yunyaoli** [comment]
@arademaker Can we close this issue? Thanks.

**Comment by alanakbik** [comment]
Hello @arademaker are you planning to merge branch `issue-4`? We just stumbled across the problems in the Portuguese UP and it seems to fix them?

**Comment by arademaker** [comment]
Yes, I need to finish this issue. I will need 1-2 weeks for that.

**Comment by yunyaoli** [comment]
@arademaker Any updates?

**Comment by arademaker** [comment]
Sorry, I will solve it this week. But we still need to reevaluate the plans for the PR #7.

**Comment by andreabac3** [comment]
UP

**Comment by arademaker** [comment]
Sorry, can you elaborate?

**Comment by andreabac3** [comment]
@arademaker Sorry, I'm referring to the train and dev file naming problem. I only realized this after producing the dataset statistics. With my comment ("UP"), I wanted to bring to attention the fact that the names are still reversed.

I take this opportunity to thank all of you for your work.

Kind regards,
Andrea
